Can't resolve 'hastscript'

```
./node_modules/refractor/core.js
Module not found: Can't resolve 'hastscript' in ...
```

The `hastscript` package needs to be moved from `devDependencies` to `dependencies` in the package.json file

--- a/package.json
+++ b/package.json
@@ -25,7 +25,8 @@
     "lang"
   ],
   "dependencies": {
-    "prismjs": "~1.6.0"
+    "prismjs": "~1.6.0",
+    "hastscript": "^3.1.0"
   },
   "devDependencies": {
     "alpha-sort": "^2.0.1",
@@ -37,7 +38,6 @@
     "chalk": "^2.0.0",
     "detab": "^2.0.0",
     "esmangle": "^1.0.0",
-    "hastscript": "^3.1.0",
     "is-hidden": "^1.1.0",
     "mdast-zone": "^3.0.1",
     "not": "^0.1.0",

--- a/package.json
+++ b/package.json
@@ -25,8 +25,8 @@
     "lang"
   ],
   "dependencies": {
-    "prismjs": "~1.6.0",
-    "hastscript": "^3.1.0"
+    "hastscript": "^3.1.0",
+    "prismjs": "~1.6.0"
   },
   "devDependencies": {
     "alpha-sort": "^2.0.1",

